pkg/webui/console/components/rights-group/index.js

```diff
@@ -42,8 +42,12 @@ const computeProps = function(props) {
   const { value, pseudoRight: grantablePseudoRight, rights: grantableRights } = props
 
   // Extract the pseudo right from own rights or granted rights
-  const derivedPseudoRight =
-    grantablePseudoRight || value.find(right => right !== RIGHT_ALL && right.endsWith('_ALL'))
+  let derivedPseudoRight
+  if (grantablePseudoRight && !Array.isArray(grantablePseudoRight)) {
+    derivedPseudoRight = [grantablePseudoRight]
+  } else if (Boolean(grantablePseudoRight) && Array.isArray(grantablePseudoRight)) {
+    derivedPseudoRight = value.filter(right => right !== RIGHT_ALL && right.endsWith('_ALL'))
+  }
 
```

Comment:
Note that `derivedPseudoRight` is also being used as `name` prop in the checkbox, which will hit prop-type validation when it is an array. (This is intersecting with #1466) 
It might also be clearer to rename it to `derivedPseudoRights` then.

Reply:
The refactor here also leads to issues in the application collaborator section:
```
index.js?0be7:65 Uncaught TypeError: Cannot read property 'includes' of undefined
    at eval (index.js?0be7:65)
    at Array.filter (<anonymous>)
    at computeProps (index.js?0be7:65)
    at withComputedProps.render (with-computed-props.js?ec96:27)
``
Happening when trying to view the application collaborator sub-view as collaborator with some individual rights granted.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/1466/commits/260c7aba4984c59b5020549d1767638f7bad00d2